packages/migrate/src/utils/unixSocket.ts

```diff
@@ -0,0 +1,17 @@
+import { DatabaseCredentials } from '@prisma/internals'
+
+const simpleUnixPathPattern = /^\.{0,2}\//
```

Comment:
A comment has been added to this regex. 

Note: I'm explicitly ignoring relative paths that do not begin with `./`. This may be problematic for some. I'm not sure. In theory, I could look for anything that includes a `/` and interpret it as a unix path which would allow someone to omit a `./` prefix when supplying a relative path.